### PR TITLE
Add QGSS22 notebooks

### PR DESCRIPTION
**Pull request by frankharkins**
[Link to preview](https://platypus-pr-1511.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/summer-school/2022/history-quantum-computing-motivation-quantum-simulation)

This PR adds the videos and supplementary information from the summer school on quantum simulation to the textbook.

### CODE REVIEW

File: notebooks/summer-school/2022/lec-1.ipynb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "\n",
+    "This lecture by Olivia Lanes was originally created for the 2022 Qiskit Global Summer School on Quantum Simulation.\n",
+    "\n",
+    "### Suggested reading\n",
+    "\n",
+    "- [_Introduction To Quantum and Classical Computing_](http://www.thomaswong.net/introduction-to-classical-and-quantum-computing-1e3p.pdf) by Thomas Wong\n",
+    "- [_The Strange World of Quantum Mechanics_](http://www.ba.ipcf.cnr.it/didattica/corsi/solid_state_chem/qm/The_Strange_World_of_Quantum_Mechanics_Styer.pdf) by Daniel F. Styer\n",
+    "- [_Introduction to Quantum Mechanics_](https://www.fisica.net/mecanica-quantica/Griffiths%20-%20Introduction%20to%20quantum%20mechanics.pdf) by David J. Griffiths\n",
+    "- _Learn Quantum Computation Using Qiskit:_\n",
+    "  - [_1.1 Introduction_](https://qiskit.org/textbook/ch-states/introduction.html)\n"
```
Comment (techtolentino) [inline notebooks/summer-school/2022/lec-1.ipynb:26]: Out of curiosity, should this link to the Textbook beta page [here](https://learn.qiskit.org/course/ch-states/introduction)?

Same question for other links in the Suggest Reading of different pages... will we eventually start routing users to the same content in Textbook beta, or for now do we still want them heading back to the old Textbook?

**Review by techtolentino — COMMENTED** [review]
Thanks for this @frankharkins 

Couple of questions:
1) I had a general question about whether or not suggest reading links should point to the Textbook beta, or the old version... 

2) Do we want this course / chapters to appear in the `Browse all content` mega dropdown menu? If so, we'll need to update this [megaMenuLinks.ts](https://github.com/Qiskit/platypus/blob/fh-qgss-22/frontend/vue/components/constants/megaMenuLinks.ts) file as well.


Thanks again!

File: notebooks/summer-school/2022/lec-1.ipynb (hunk 1 above)
Comment (frankharkins) [reply notebooks/summer-school/2022/lec-1.ipynb]: Good catch, yes I think it's probably best we point to the new textbook as some of the old pages are outdated now. I've fixed this in [aa5405e](https://github.com/Qiskit/platypus/pull/1511/commits/aa5405e67a39149c79fac656f6102d4d7a0aa531), but left the link in the lab1 notebook, as I feel that's more archival than a living reference.

**Comment by frankharkins** [comment]
@techtolentino thanks for reviewing! I forgot about the mega menu, but looking now I don't think the other summer schools are included 🤷 . Not sure if that's an error or there was a conscious decision not to include summer schools.

**Comment by techtolentino** [comment]
> @techtolentino thanks for reviewing! I forgot about the mega menu, but looking now I don't think the other summer schools are included 🤷 . Not sure if that's an error or there was a conscious decision not to include summer schools.

@frankharkins - you're right about that... it's okay if we leave them out for now; we can wait for Russell to get back and open a new issue/PR if we find out they should be included.

Thanks!

**Review by techtolentino — APPROVED** [review]
I'm good w/ this PR, thank you @frankharkins 

BTW, for the dropdown megamenu, I created [this issue](https://github.com/Qiskit/platypus/issues/1584) for us to discuss w/ Russell when he returns.


👏🏼 nicely done here

**Review by vabarbosa — APPROVED** [review]
@frankharkins thank you!  i am approving but i have a non-blocking comment or something to think about.

if i am not mistaken the four labs are the challenges from the summer school... not advocating for it but was it considered to have the labs rendered and allow users to go through them and be able to grade them from the site (instead of having to download it)?

learning platform is using same grading service as challenge and we already have all the validation code. the work that would need to be done is updating the individual grading cells to include the appropriate metadata.

if we prefer users download them, maybe we should [point to instructions for running the grader locally](https://github.com/qiskit-community/Quantum-Challenge-Grader#run-locally) so they can go through the labs & have their solutions validated

**Pull request merged**